tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "outDir": "./src/",
+    "sourceMap": true,
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "ES5"
+  }
+}
+
+    // "include": ["./src/**/*"]
```

Comment:
Why did you comment this out? I think you might need something like 
`"include": ["./src/**/*/.ts`"